tests/base.py

```diff
@@ -99,6 +100,12 @@ def setUp(self):
             resource = Resource(name='Markers',
                                 quantity=3)
             resource.save()
+            resource_2 = Resource(name='Pen',
```

Comment:
Nice work @jesseinit . I have a small concern though. Is it possible to just use the `Markers` resources instead of adding a new one.

Reply:
I tried that approach before arriving at this. The issue is that it was breaking a test case for the `assign room` feature and also another test for the `get resource list` feature. 
I'd appreciate a better way to going about it if you've got any.